### DISCUSSION TITLE
How to modify the source code to automatically start the MCC built-in command after logging into the server

### DISCUSSION THREAD

**Discussion by FIZxingchen** [ORelio/Minecraft-Console-Client#2561] [Q&A]
我想在mcc的源代码部分修改，使其能在登陆服务器后自动/l password,(这个可以用SendText("/l password")解决,可是我并不知道怎么启动mcc内置命令/changeslot和/useitem,因为SendText("/changeslot 3")只会发送到服务器而不是mcc,我加入的服务器需要登陆服务器后再次输入密码并与方块交互

希望有大佬帮我看看怎么解决,太谢谢了

I want to modify the MCC source code so that it can automatically/L password, (this can be done with SendText ("/L password") , but I don't know how to start the MCC built-in commands/Changeslot and/USEITEM, because SendText ("/changeslot 3") will only be sent to the server and not to the MCC, and the server I join needs to log in to the server and enter the password again and interact with the box.)

I hope I have a big help to see how to solve, thank you very much

^*^我已阅读mcc开源规则,I read MCC open source rules

**Comment by ReinforceZwei** [comment]
Write a [script ](https://mccteam.github.io/guide/creating-text-script.html) and use [script-scheduler](https://mccteam.github.io/guide/chat-bots.html#script-scheduler) to run the script with `Trigger_On_Login = true`

**Reply by FIZxingchen** [reply]
我有些不理解(I don't understand)

**Comment by FIZxingchen** [comment]
是要在脚本添加Trigger_On_Login = true吗(To add Trigger_On_Login = true to the script ?)
